### ISSUE TITLE
Cannot set time with bigBanner = false

### ISSUE THREAD

**Issue by Tododispositivos** [CuppaLabs/angular2-datetimepicker#7]
Cannot set time with bigBanner = false
![captura de pantalla 2017-12-26 a la s 00 21 54](https://user-images.githubusercontent.com/15638522/34346064-d5e2a83a-e9d2-11e7-99c5-e4d804e5e2fb.png)

**Comment by Nasty94** [comment]
it would great to have this option without bigBanner

**Comment by tburger** [comment]
I agree.  It would be nice to have time below the date when timepicker is enabled

**Comment by chandesh07** [comment]
`<div class="clock-icon" *ngIf="!settings.bigBanner">
            <i class="fa fa-clock-o" aria-hidden="true" (click)="timeView = !timeView"></i>
        </div>`
I checked the file in my node modules, seem this code is missing when we install the plugin

**Comment by kingjulian32** [comment]
does this control is still maintenance/improved?

**Comment by HassanMoin** [comment]
+1
